src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/client/model/prisoner/CurrentlyManaged.kt

```diff
@@ -0,0 +1,26 @@
+package uk.gov.justice.digital.hmpps.personrecord.client.model.prisoner
+
+data class CurrentlyManaged(val status: Boolean?) {
+
+  companion object {
+    private const val EMPTY_STATUS = ""
+
+    private enum class Status(val status: String) {
+      ACTIVE_IN("Active : IN"),
+      ACTIVE_OUT("Active: OUT"),
+      INACTIVE_TRANSFER("Inactive: TRN"),
+      INACTIVE_OUT("Inactive: OUT"),
+    }
+
+    fun from(inputStatus: String? = EMPTY_STATUS): CurrentlyManaged {
+      val status: Boolean? = when (inputStatus) {
+        Status.ACTIVE_IN.status -> true
+        Status.ACTIVE_OUT.status -> true
+        Status.INACTIVE_TRANSFER.status -> true
+        Status.INACTIVE_OUT.status -> false
```

Comment:
what is the difference between `false` and `null`?

Reply:
I believe as its only for prisoner reocrds we dont wont the rest of the records to hold any value as per the AC's on: [this](https://dsdmoj.atlassian.net/browse/CPR-440)

Reply:
I am questioning the ACs themselves. What information do we gain from `null`?

Also `currently managed`is a concept in probation as well as prison (naturally enough).
Are we ever going to store that?

Reply:
Yeah there was a discussion around this. I dont recall most of it but we know for sure with a certain level for prison that they are within our care. But under probation is subjective. Probably worth asking Tony to clarify this.